package.json

```diff
@@ -9,6 +9,8 @@
     "big.js": "^6.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router": "^6.10.0",
+    "react-router-dom": "^6.10.0",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
```

Comment:
- [ ] Kindly remove line 17 here in the `package.json` file this one:

![image](https://user-images.githubusercontent.com/68796232/229566856-e3536b03-00b6-4863-af06-173aec59e7d7.png)

 because it directs to a blank page when you run the dev server. When I removed it everything works just fine.
 
 
![image](https://user-images.githubusercontent.com/68796232/229566889-004b72c6-cd9f-4405-8dee-099b674c5667.png)